src/FixedPointDecimals/FixedPointDecimals.jl

```diff
@@ -220,17 +220,10 @@ realmin{T <: FD}(::Type{T}) = eps(T)
 realmax{T <: FD}(::Type{T}) = typemax(T)
 
 # printing
-function show{T}(io::IO, x::FD{T, 0})
-    iscompact = get(io, :compact, false)
-    if !iscompact
-        print(io, "FixedDecimal{$T,0}(")
-    end
+function print{T}(io::IO, x::FD{T, 0})
     print(io, x.i)
-    if !iscompact
-        print(io, ')')
-    end
 end
-function show{T, f}(io::IO, x::FD{T, f})
+function print{T, f}(io::IO, x::FD{T, f})
     iscompact = get(io, :compact, false)
```

Comment:
For fixed point decimals I do like having the trailing zeros displayed. Additionally the help for `print` states:

> Write (to the default output stream) a canonical (un-decorated) text
  representation of a value if there is one, otherwise call show.

Since the `show` method defaults to showing trailing zeros I think that `print` should display the same information as `show` but without the type information. I think whatever is decided should be the default for both `show` and `print`.